TACC/protx-dashboard
# Task/cooks 357 remove hover trace text on analysis chart

## Overview: ##
Remove hover trace labels on analytics chart

## Related Jira tickets: ##

* [COOKS-357](https://jira.tacc.utexas.edu/browse/COOKS-357)

## Summary of Changes: ##
Set hoverinfo to skip for scatter and histogram analytics chart 

## Testing Steps: ##
1. Go to [https://cep.test/protx/dash/analytics](https://cep.test/protx/dash/analytics)
2. Hover over state wide histogram, specifically around the vertical lines and make sure trace labels no longer appear
3. Click on any county and try the same

## UI Photos:

Before changes:
<img width="1531" alt="Screen Shot 2022-11-22 at 9 59 21 AM" src="https://user-images.githubusercontent.com/96220951/203361621-d68426c6-9455-4f98-84a4-77700a151154.png">

After changes:

<img width="1536" alt="Screen Shot 2022-11-22 at 10 01 42 AM" src="https://user-images.githubusercontent.com/96220951/203362303-26d47882-e3cd-43db-aef3-ef839b3412af.png">



## Notes: ##
